lib/apiservers/engine/backends/container.go

```diff
@@ -478,50 +503,35 @@ func (c *Container) containerStart(name string, hostConfig *container.HostConfig
 	}
 
 	// get a handle to the container
-	var getRes *containers.GetOK
-	getRes, err = client.Containers.Get(containers.NewGetParamsWithContext(ctx).WithID(name))
+	handle, err := c.Handle(name, paramName)
```

Comment:
Not sure I understand your comment. We are still doing what the previous code is doing. I just created a function https://github.com/vmware/vic/pull/1918/files#diff-6e989557b980355774bd4187364245fbR138 to make it DRY